packages/libsql/src/lib/Category.ts

```diff
@@ -252,18 +253,25 @@ class InternalCategory<Event, State, Context>
   }
 
   async reload(streamId: Equinox.StreamId, _requireLeader: boolean, t: TokenAndState<State>) {
-    if (this.access.type === "RollingState") {
-      return this.load(streamId, 0)
+    switch (this.access.type) {
+      // These strategies all have it in common that their state 
+      // can be reconstituted in a single round trip to the database
+      // as such is it as cheap to load them as it is to reload them
```

Comment:
```suggestion
      // as such, is it as cheap to load them as it is to reload them
```
But maybe add even more explicit stuff?
// for this reason, unlike with some other stores, there's no particular win to be had in terms of
// using a more effficient index based on some cached token or etag, reducing roundtrips etc.